src/components/v5/shared/RichTextDisplay/RichTextDisplay.tsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/no-this-in-sfc */
+import Blockquote from '@tiptap/extension-blockquote';
+import Bold from '@tiptap/extension-bold';
+import Document from '@tiptap/extension-document';
+import Heading from '@tiptap/extension-heading';
+import Text from '@tiptap/extension-text';
+import Underline from '@tiptap/extension-underline';
+import { EditorContent, useEditor } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import React from 'react';
+
+const displayName = 'v5.RichTextDisplay';
+
+interface RichTextDisplayProps {
+  content: string;
+}
+
+const RichTextDisplay = ({ content }: RichTextDisplayProps) => {
+  const editor = useEditor(
+    {
+      editable: false,
+      extensions: [
+        Document,
+        Text,
+        Underline,
+        StarterKit.configure({
+          heading: false,
+        }),
+        Heading.configure({ levels: [1, 2, 3] }).extend({
+          levels: [1, 2],
+          renderHTML({ node, HTMLAttributes }) {
+            // @ts-ignore
+            const level = this.options.levels.includes(node.attrs.level)
+              ? node.attrs.level
+              : // @ts-ignore
```

Comment:
Could you add some comments explaining why are we ignoring ts errors  here?

Reply:
ah sorry I have no idea why we do this, I just moved this component but Git thinks I created it :D I'll check it out though